motion-1/Overview.bs

```diff
@@ -608,7 +608,7 @@ how to process an 'offset-rotation'.
 <h3 id="offset-shorthand">Offset shorthand: The 'offset' property</h3>
 <pre class='propdef'>
 	Name: offset
-	Value: &lt;'offset-path'> && &lt;'offset-distance'> && &lt;'offset-position'> && &lt;'offset-anchor'> && &lt;'offset-rotation'>
+	Value: [ <<offset-position>> <<offset-rotation>>? <br> | <<offset-position>>? <<offset-path>> [ <<offset-distance>> || <<offset-rotation>> ]? ] <br> [ / <<offset-anchor>> ]?
```

Comment:
Currently I'm leaning towards
```
[<offset-position>]? [<offset-path> [<offset-distance> || <offset-rotation>]? ]? [ / <offset-anchor> ]?
```
as being closest to what the WG agreed on while also avoiding the issue of offset-position and offset-rotation being ambiguous when adjacent.

Another option that would also probably work is
```
[<offset-position> / ]? [<offset-path> || <offset-distance> || <offset-rotation>]? [ / <offset-anchor> ]?
```
However I think we'd want to modify this so that `<offset-position> / <offset-anchor>` is also valid, and I'm not sure that freeing up the position of offset-path is worth the extra complexity.